pkg/reconcilermanager/controllers/reposync_controller.go

```diff
@@ -324,6 +360,7 @@ func (r *RepoSyncReconciler) handleReconcileError(ctx context.Context, err error
 			logFieldObjectRef, opErr.ID.ObjectKey.String(),
 			logFieldObjectKind, opErr.ID.Kind,
 			logFieldOperation, opErr.Operation)
+		reposync.SetReconciling(rs, stage, fmt.Sprintf("%s stalled", stage))
```

Comment:
Curious why are we setting the reconciling condition in these areas now?

Reply:
It seemed like a good idea to convey that it was Reconciling that was Stalled and that Reconciling was still not complete because it was Stalled. Otherwise, it's not super clear what Stalled means.

AFAICT, everything that knows to look for Stalled and Reconciling (kstatus, nomos status) looks for Stalled first, so it shouldn't cause any problems. I should probably verify with the UI team tho that it won't break anything there.